File: program126.java
import java.util.*;
class ArrayProgram
{
    public void Display(int [] Arr, int iSize)
    {
        int iCnt = 0;
        System.out.println("Entered  elements are :");

        for(iCnt = 0; iCnt < iSize ; iCnt++)
        {
            System.out.println(Arr[iCnt]);
        }

    }
}

class program126
{
    public static void main(String A[])
    {
        int iCnt = 0, iLength = 0;

        Scanner sobj = new Scanner(System.in);

        System.out.println("Enter number of elements ");
        iLength = sobj.nextInt();

        int [] Arr = new int [iLength];

        System.out.println("Enter elements :");

        for(iCnt = 0; iCnt < iLength ; iCnt++)
        {
            Arr[iCnt] = sobj.nextInt() ;
        }

        ArrayProgram aobj = new ArrayProgram();
        aobj.Display(Arr,iLength);

        

    }
}

File: C++/program126.cpp
#include<iostream>
using namespace std;

void Display(int Arr[], int iSize)
{
    int iCnt = 0;
    cout<<"Elements of array are "<<'\n';

    for(iCnt = 0; iCnt < iSize; iCnt++)
    {
        cout<<Arr[iCnt]<<'\n';
    }
}

int main()
{
    int iLength = 0, iCnt = 0;
    int *ptr = NULL;

    cout<<"Enter number of elements "<<'\n';
    cin>>iLength;

    ptr = new int[iLength];

    if(NULL == ptr)
    {
        cout<<"Unable to allocate memory"<<'\n';
        return -1;
    }

    cout<<"Enter array elements "<<'\n';
    for(iCnt = 0; iCnt < iLength; iCnt++)
    {
        cin>>ptr[iCnt];
    }

    Display(ptr,iLength);

    delete [] ptr;
    return 0;
}